cliff.toml

```diff
@@ -65,6 +65,7 @@ split_commits = false
 # regex for preprocessing the commit messages
 commit_preprocessors = [
   { pattern = '\((\w+\s)?#([0-9]+)\)', replace = "([#${2}](<REPO>/issues/${2}))" },
+  { pattern = '.*', replace_command = 'typos --write-changes -' },
```

Comment:
Wow, this is really clever.  However it means that it will run `typos` for each commit thus it takes a bit time for me to generate the changelog. But I guess it is a good tradeoff for the sake of having an example of this cool trick.

One thing that we're missing is a comment that is explaining what is done here and linking to the `typos` repository 🐻

Reply:
That said, I'm now wondering if it's such a clever trick. If you try this, you'll understand why

```
echo "overrided" | typos --write-changes -
```

when 2 different fixes are possible, `typos` just bail out. It's not even putting the message on `stderr` and the word on `stdout` (which might have help to, at least, keep the word as-is). What's your take about that?

Reply:
At the very least, I added a comment for the time being in [1c98c8f](https://github.com/orhun/git-cliff/pull/316/commits/1c98c8f4c9b64d05d569f4ae2caebd6b411656ad).

Reply:
I just pushed 205cdbb391b03244abaf2e5e0651976544c78ff4, it should be fine now.

Reply:
Oh sweet idea. Thank you.